src/test/fuzz/node_eviction.cpp

```diff
@@ -40,7 +40,7 @@ FUZZ_TARGET(node_eviction)
     // Make a copy since eviction_candidates may be in some valid but otherwise
     // indeterminate state after the SelectNodeToEvict(&&) call.
     const std::vector<NodeEvictionCandidate> eviction_candidates_copy = eviction_candidates;
-    const std::optional<NodeId> node_to_evict = SelectNodeToEvict(std::move(eviction_candidates));
+    const std::optional<NodeId> node_to_evict = SelectNodeToEvict(std::move(eviction_candidates), /*force=*/false);
```

Comment:
(I didn't test that this is okay)
```suggestion
    const std::optional<NodeId> node_to_evict = SelectNodeToEvict(std::move(eviction_candidates), /*force=*/fuzzed_data_provider.ConsumeBool());
```

Reply:
yeah thanks good diea